CHANGELOG.md

```diff
@@ -0,0 +1,14 @@
+# Upgrade to V4
+
+The URL structure has changed to make it possible to add more options. The old URL structure is not supported anymore. But can easily be converted to the new structure.
+
+Some examples:
+
+- `/scaled/500x500/foobar.jpg` -> `/scaled/width:500_height:500/foobar.jpg`
+- `/scaled/x400/foobar.jpg.webp` -> `/scaled/height:400_convert:webp/foobar.jpg.webp`
```

Comment:
I would spend one extra sentence here to describe the new format, and the fact that `webp` needs to be both in the options and as an extension.